workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layouts/1_home/page-definition.json

```diff
@@ -2728,9 +2745,11 @@
 		"type": "Root"
 	},
 	"settings": {
+		"colorSchemeName": "01",
```

Comment:
@rtienpal now that we have this template saved I try not to update the settings portion anymore, we dont need to define the these properties or want the version number to increase